src/abstractdataframe/show.jl

```diff
@@ -24,12 +24,23 @@ end
 Render a value to an IO object. Unlike
 `show`, render strings without surrounding quote marks.
 """
-ourshow(io::IO, x::Any) =
-    show(IOContext(io, :compact=>get(io, :compact, true), :typeinfo=>typeof(x)), x)
+function ourshow(io::IO, x::Any)
+    ctxt = IOContext(io, :compact=>get(io, :compact, true), :typeinfo=>typeof(x))
+    show(ctxt, "text/plain", x)
+end
+
 ourshow(io::IO, x::AbstractString) = escape_string(io, x, "")
 ourshow(io::IO, x::Symbol) = ourshow(io, string(x))
 ourshow(io::IO, x::Nothing) = nothing
 
+# https://github.com/JuliaLang/julia/pull/34730
+# if VERSION < v"1.5.0-DEV.261"
```

Comment:
I'll update this to:
```julia
if VERSION < v"1.5.0-DEV.261" || VERSION < v"1.5.0-DEV.X"
```
Once https://github.com/JuliaLang/julia/pull/34741 is merged.

Reply:
Done